frontend/src/features/dashboard/dashboardSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import UserAuthApi from '../../helpers/UserAuthApi';
+
+export const getDashboard = createAsyncThunk('dashboard/id', async () => {
+    return UserAuthApi.getAll().then(res => res.json());
```

Comment:
since you are using aysnc await function you can do it like this:

```suggestion
export const getDashboard = createAsyncThunk('dashboard/id', async () => {
    const response = await UserAuthApi.getAll();
    return response.json();
```

Reply:
Thank you Sir, I'll update this one.